### DISCUSSION TITLE
Maravilhas da Arquitetura Moderna #1

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#524] [Curiosidades]
Post de referência: https://www.vish.blog.br/curiosidades/maravilhas-da-arquitetura-moderna-1/

**Comment by one-eyed-willy** [comment]
Pior que perto de onde eu moro tem complexo de passarelas amarelas, velhas e com a pintura meio gasta que realmente parece uma pista da Hot Wheels

**Comment by angellisy** [comment]
Amei!
